CODE_OF_CONDUCT.md

```diff
@@ -63,7 +63,7 @@ The following code of conduct is based on full compliance with [ASF CODE OF COND
  - Environment preparation codes should be separate from test codes.
  - Only those that relate to junit `Assert`, hamcrest `CoreMatchers` and `Mockito` can use static import.
  - For single parameter asserts, `assertTrue`, `assertFalse`, `assertNull` and `assertNotNull` should be used.
- - For multiple parameter asserts, `assertThat` should be used.
+ - For multiple parameter asserts, `org.hamcrest.MatcherAssert.assertThat` should be used.
```

Comment:
1, It might be better to put `MatcherAssert` in `Only those that relate to junit Assert, hamcrest CoreMatchers and Mockito can use static import.`, and keep `assertThat` simple. `org.junit.Assert.assertThat` is deprecated now, contributors will notice it in IDE.

2, Looks there's another `code.en.md` could be updated.

Reply:
I tweaked the word order to make it readable.

Reply:
Nice